### PR TITLE
ENH add libibcm and libibverbs

### PR DESCRIPTION
**Pull request by beckermr**
Please see the repo readme for directions on how to make PRs on this repo.

Checklist:

- [ ] if you have added a CDT, it appears in the `cdt_slugs.yaml` file
- [ ] if you have changed the CDT generator code (`rpm.py`), you have bumped
  the build number in `conda_build_config.yaml` and have remade all of the
  recipes via running `python gen_cdt_recipes.py`
- [ ] if you have added a custom CDT recipe, you have added the name of the CDT
  with `custom: true` in the `cdt_slugs.yaml` file.
- [ ] all CDT recipes have build number set by `{{ cdt_build_number }}` for
  old-style/legacy CDTs or `{{ cdt_build_number|int + 1000 }}` for new-style CDTs
- [ ] if you see a warning about a CDT not having a license, you have added the
  `license_file` key in the `cdt_slugs.yaml` file with the path to the appropriate
  license in `licenses/`

### CODE REVIEW

**Comment by jakirkham** [comment]
Thanks Matt! Is this good to merge or do we need to wait for something?

**Comment by beckermr** [comment]
I need to leave issues on the feedstocks and archive them. Will do later.

**Comment by jakirkham** [comment]
Done. Anything else?

File: cdt_slugs.yaml [hunk 1]
```diff
@@ -34,12 +34,32 @@ chkconfig:
...
+    - cos7-x86_64
+    - cos7-aarch64
+    - cos7-ppc64le
+  license_file: licenses/libibcm-devel-license
+libibverbs:
+  custom: false
+  skipped_cdts:
+    - cos7-x86_64
+    - cos7-aarch64
+    - cos7-ppc64le
```
Comment (beckermr) [inline cdt_slugs.yaml:55]: @jakirkham It appears they are being skipped on cos7. You'll need to remove that.

Comment (jakirkham) [reply cdt_slugs.yaml:55]: Thanks Matt. Doing in PR ( https://github.com/conda-forge/cdt-builds/pull/26 ) 🙂

**Pull request merged**